tests/wallet3_tests/mock_wallet.hpp

```diff
@@ -74,6 +74,11 @@ class MockWallet : public Wallet
       db->store_transaction(hash, height, dummy_outputs);
       db_tx.commit();
     };
+
+    ~MockWallet()
+    {
+    }
```

Comment:
The destructor here isn't needed with the base class now having a virtual destructor.